install.sh

```diff
@@ -77,26 +77,50 @@ process_arguments() {
 verify_configuration_file() {
   log "Verifying the config file."
 
+  HAS_VALIDATION_ERR=false
```

Comment:
```suggestion
  HAS_VALIDATION_ERR=
```

Reply:
I am sorry, that is my mistake. It should be:

```
HAS_VALIDATION_ERR="false"
```
or we can have 
```
HAS_VALIDATION_ERR="yes"
```

I think the scripts are hard to read and using the `1` doesn't really help. There are other options than using numbers. https://stackoverflow.com/a/47092826

Sorry @yzha645 for leading you astray.

Reply:
Using `1` and `0` for flags is used widely in the scripts so using this will keep consistency. 
We can refactor this later when we will add other products if is needed.